src/common/ceph_mutex.h

```diff
@@ -107,6 +107,47 @@ namespace ceph {
   #define ceph_mutex_is_not_locked_by_me(m) (!(m).is_locked_by_me())
 }
 
+#elif __MINGW32__
```

Comment:
What about `CEPH_DEBUG_MUTEX` case?  I think it's built on pthread API, are we accepting it to be broken on Windows?

Reply:
I think it’s still usable for debug purposes. For now, we can add a note about the known Windows issues.